sharktank/sharktank/models/punet/tools/import_brevitas_dataset.py

```diff
@@ -113,6 +122,15 @@ def _get_json_tensor(
     if input_zp is not None:
         assert torch.count_nonzero(input_zp) == 0
 
+    if output_scale is not None:
+        output_quantizer = StaticScaledQuantizer(
+            name=f"{layer_name}.q_output",
+            scale=1.0 / output_scale,
+            reciprocal_scale=output_scale,
+            dtype=torch.float8_e4m3fnuz,
```

Comment:
Add a TODO to get the output dtype from the json file (somehow).

Reply:
TODO added